python/state_space_estimation.py

```diff
@@ -82,19 +79,25 @@
 elif source == 'real':
     data = pd.read_csv('../data/real_data.csv', index_col='DATE')
 
+elif source == 'integration_test':
+    raise NotImplementedError("Lets have a really small dataset for integration testing")
+
 else:
     raise ValueError("Source data not supported")
 
 
-def test(data, alpha):
+def test(data, alpha, debug=False):
```

Comment:
Should this method move into a library?
Here's one place a docstring would definitely have helped me